KosherCocoa/Library/Core/Solar/KCAstronomicalCalendar.h

```diff
@@ -24,11 +24,10 @@ NS_SWIFT_NAME(AstronomicalCalendar)
 /**
  * The calculator to use. 
  *
- *  In KosherCocoa, it's always a KCSunriseAndSunsetCalculator. The
- *  original Java library offered several calculator options.
+ *  The default calculator is the KCNOAACalculator. However, you can use the KCSunriseAndSunsetCalculator as well.
  */
 
-@property (nonatomic, strong, nonnull) KCSunriseAndSunsetCalculator *astronomicalCalculator;
+@property (nonatomic, strong, nonnull) KCNOAACalculator *astronomicalCalculator;
```

Comment:
Change the type to `id<KCAstronomicalCalculator>`, to specify that the calculator can be an object of either type which conforms to the `KCAstronomicalCalculator` protocol. We also want the imports for both calculator classes + the protocol header.

Reply:
We'd also need to change the type in the KCAstronomicalCalendar.m file

Reply:
Let's do it.

Reply:
I tried doing it the way you said with `id<KCAstronomicalCalculator>` but the compiler was giving me issues with it. I used the `typeof` keyword instead and it worked.